phpcs-ruleset.xml

```diff
@@ -81,6 +81,9 @@
         <exclude-pattern>profiles/common/modules/custom/tmgmt_poetry/inc/tmgmt_poetry.webservice.inc</exclude-pattern>
     </rule>
 
+    <!-- Chosen WSDL callback function FPFISPoetryIntegrationRequest do not follow camel case format. -->
```

Comment:
Here we are excluding the whole PoetryCallback.php file while we should only exclude that particular method signature by using partial ignore as the example below:

```
$xmlPackage = new XMLPackage;
// @codingStandardsIgnoreStart
$xmlPackage['error_code'] = get_default_error_code_value();
// @codingStandardsIgnoreEnd
$xmlPackage->send();
```

See https://pear.php.net/manual/en/package.php.php-codesniffer.advanced-usage.php for more info.